pgx_async/bin/pgx_async_example.ml

```diff
@@ -169,7 +169,7 @@ let main () =
        >>| fun x -> printf !"%{sexp: Pgx.row list list}\n" x
     )
 
-let () = 
+let () =
   let summary = "example command" in
-  Command.async ~summary Command.Spec.empty main
+  Command.async_spec ~summary Command.Spec.empty main
```

Comment:
what's this change for?

Reply:
Forgot to add to the description since it seems like jbuilder doesn't build this file and it only cropped up in godzilla. But the command interface was changed: https://ocaml.janestreet.com/ocaml-core/v0.10/doc/core/Core/Command/

See `basic_spec` vs `basic`. The one we were using before was renamed to `*_spec`. In godzilla I was just renaming the call sites but since this is a smaller project, I can be less lazy and convert the spec to use the new "nicer" format.